afs/afs-ws/afs-ws-storage/src/main/java/com/powsybl/afs/ws/storage/NodeEventClient.java

```diff
@@ -52,8 +69,52 @@ public void onError(Throwable t) {
         }
     }
 
+
     @OnClose
-    public void onClose(Session session) {
-        LOGGER.trace("Node event websocket session '{}' closed for file system '{}'", session.getId(), fileSystemName);
+    public void onClose(Session session, CloseReason closeReason) {
+
+        if (closeReason.getCloseCode().getCode() == 1006) {
```

Comment:
Is there an existing constant instead of 1006? What this error code means? Is there other error codes you should manage?

What do we do if the error code is not 1006